### DISCUSSION TITLE
Multi year trends

### DISCUSSION THREAD

**Discussion by redge76** [hargata/lubelog#716] [General]
Hi,

Is there a way to have a multi-year graph for the dashboard data?
Currently, it seems that the "all-time" option displays a monthly average. This allows me to answer questions like: Is my car more expensive in winter or summer?
However, there isn’t an "unaggregated all-time" option to display the trend of my car expenses since I bought it. For example, with inflation, I suspect my expenses have increased, but is that really the case? Also, is my mileage per year/month increasing over time?

**Comment by hargata** [comment]
Next Release:

![monthlyyearlycost](https://github.com/user-attachments/assets/52c0c58e-8090-4191-ad81-e31074b52f1b)

Clicking on the table headers toggles between Cost, Distance Traveled, and Cost per Distance Traveled.

**Comment by hargata** [comment]
Released
